lib/request_handler/page_parser.rb

```diff
@@ -93,7 +98,9 @@ def apply_max_size_constraint(size, prefix)
 
     def lookup_nested_config_key(key, prefix)
       key = prefix ? "#{prefix}.#{key}" : key
-      config.lookup!(key)
+      lookup!(config, key)
```

Comment:
we could probably add a `lookup` method to the config helper. Then we don't need the error handling here and could also use `lookup` in other places where we currently use `dig`